span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/ExcludeSpanRuleEvaluator.java

```diff
@@ -0,0 +1,181 @@
+package org.hypertrace.core.spannormalizer.jaeger;
+
+import static org.hypertrace.core.span.constants.v1.Http.HTTP_REQUEST_URL;
+import static org.hypertrace.core.span.constants.v1.OTSpanTag.OT_SPAN_TAG_HTTP_URL;
+import static org.hypertrace.core.spannormalizer.jaeger.JaegerSpanNormalizer.OLD_JAEGER_SERVICENAME_KEY;
+
+import io.jaegertracing.api_v2.JaegerSpanInternalModel;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import org.apache.commons.lang3.StringUtils;
+import org.hypertrace.core.datamodel.AttributeValue;
+import org.hypertrace.core.semantic.convention.constants.http.OTelHttpSemanticConventions;
+import org.hypertrace.core.span.constants.RawSpanConstants;
+import org.hypertrace.core.span.constants.v1.Http;
+import org.hypertrace.core.spannormalizer.util.JaegerHTTagsConverter;
+import org.hypertrace.span.processing.config.service.v1.ExcludeSpanRule;
+import org.hypertrace.span.processing.config.service.v1.Field;
+import org.hypertrace.span.processing.config.service.v1.LogicalOperator;
+import org.hypertrace.span.processing.config.service.v1.LogicalSpanFilterExpression;
+import org.hypertrace.span.processing.config.service.v1.RelationalOperator;
+import org.hypertrace.span.processing.config.service.v1.RelationalSpanFilterExpression;
+import org.hypertrace.span.processing.config.service.v1.SpanFilter;
+
+public class ExcludeSpanRuleEvaluator {
```

Comment:
I can see, there are 3 types of exclusions
- Tenant ID based filtering https://github.com/hypertrace/hypertrace-ingester/blob/main/span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/TenantIdHandler.java#L30 (_I don't think it's been in use anymore_)
- Span Filter
- Exclude Rules

We should have a single orchestrator, driving these 3 exclusions, with a TODO comments to get rid of tenant id based exclusion and span filter based exclusion eventually